lib/vanagon/platform/rpm.rb

```diff
@@ -40,6 +49,13 @@ def output_dir(target_repo = "products")
         super
       end
 
+      # Method to derive the directory for source artifacts
+      #
+      # @param target_repo [String] repo the source artifacts are targeting
+      def source_dir(target_repo = "products")
+        @source_dir ||= File.join(@os_name, @os_version, target_repo, 'SRPMS')
```

Comment:
No, this is happening before those vanagon changes. This should get updated when those changes go in.